ds701_book/jupyter_notebooks/19-Regression-III-More-Linear.ipynb

```diff
@@ -7,39 +7,27 @@
         "---\n",
```

Comment:
<p>I think $\hat{\boldsymbol{\beta}} = (X^TX +cI)^{−1}X^T$ is <strong>not</strong> a linear function of X.  </p><p> </p><p>Regarding Scaling -- should we point out that we said in the linear regression lecture that scaling was <strong>not</strong> required. But now for this method, it <strong>is</strong> required again.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tools4ds/DS701-Course-Notes/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Removed linear function.</p><p> </p><p>Added a comment about scaling as an aside.</p>